ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -80,6 +80,15 @@ then
 	set AUTOCNF no
 else
 	set AUTOCNF yes
+	param set SYS_AUTOCONFIG 1
+fi
+
+if param compare SYS_AUTOCONFIG 1
+then
+	set AUTOCNF yes
+
+	# Wipe out params except RC*, flight modes, total flight time, accel cal, gyro cal, next flight UUID
+	param reset_nostart RC* COM_FLTMODE* LND_FLIGHT_T_* TC_* CAL_ACC* CAL_GYRO* COM_FLIGHT_UUID
```

Comment:
Resetting all params is not great, if you have certain config options you want to keep.

Reply:
True, but keeping params is not great either. Let's imagine you first simulate airframe 1 which set some non-default gains, then you switch to airframe 2 which does not set these gains. The reason why airframe 2 does not work is not obvious.
An alternative solution that I would like would be to have a "virtual board" created for each sitl airframe in which all params and logs are kept separate. But I do not know how much that would represent.
In the meantime, this PR makes sitl behave the same way as a physical board.

Reply:
Yes I'm aware.
We can use a param file name that contains the model id:
https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d-posix/rcS#L60
You would need to add `param select eeprom/parameters_"$REQUESTED_AUTOSTART"` before.

Reply:
This sounds feasible 👍

Reply:
done